doc/source/drivers/vector/pgdump.rst

```diff
@@ -182,6 +182,13 @@ The following layer creation options are supported:
       DROP TABLE from being emitted. Set to IF_EXISTS
       in order DROP TABLE IF EXISTS to be emitted (needs PostgreSQL >= 8.2)
 
+-  .. config:: OGR_PG_SKIP_CONFLICTS
```

Comment:
I understand the symmetry with the PG driver, but it would be best that this is dealt as a SKIP_CONFLICTS layer creation option to avoid proliferation of global configuration option

Reply:
That makes sense. I've felt the pain point in needing to use a mix of `--config` and `-lco` in a command.

It seems like such an option should be supported for the PG driver too, or would that be unnecessary?

Reply:
> or would that be unnecessary?

would be nice, but not a requirement for the scope of this PR